src/panel/poi/PoiPanel.jsx

```diff
@@ -192,7 +192,9 @@ export default class PoiPanel extends React.Component {
     const uri = `/places/${params}`;
 
     Telemetry.add(Telemetry.POI_BACKTOLIST);
-    fire('restore_location');
+    if (!window.no_ui) {
+      fire('restore_location');
+    }
```

Comment:
I am not sure to understand how this change is supposed to solve the problem you are describing.  
This function `backToList` is used only when clicking on the button at the top of the PoiPanel after an item has been selected from a list. Or am I missing something ?

Reply:
Ah, let me check, there may be a mistake as there are 2 different "getLastLocation()" functions in the project =_=